src/components/Chat/Chat.tsx

```diff
@@ -815,6 +836,63 @@ export class Chat extends React.Component<ChatProperties, any> {
             </div>
         );
     }
+
+    display_details = (event) => {
+        // if (this.props.nolink || !(this.state.user.id || this.state.user.player_id) || this.state.user.anonymous || (this.state.user.id || this.state.user.player_id) < 0) {
+        //     return;
+        // }
+
+        if (!this.props.fakelink && shouldOpenNewTab(event)) {
+            /* let browser deal with opening the window so we don't get the popup warnings */
+            return;
+        }
+
+        // event.stopPropogation();
+        // event.preventDefault();
+
+        let channn = global_channels.map((chan) => getChannel(chan.id).id = chan.id);
+        console.log("chan id " + channn);
+        // console.log("chan id " + channn.id);
+        let ggg = this.state.group_channels.map((g) => getChannel("group-" + g.id).id = g.id);
+        console.log("group id " + ggg);
+        console.log("group id " + ggg.id);
+        let ttt = this.state.tournament_channels.map((t) => getChannel("tournament-" + t.id).id = t.id);
+        console.log("tournament id " + ttt);
+        console.log("tournament id " + ttt.id);
+        // let aaa = this.state.active_channel.map((a) => getChannel(a.id).id = a.id);
+        let aaa = this.state.active_channel;
+        console.log("active id " + aaa);
+        // let chan = this.state.group_channels;
+        // console.log("group_details " + this.state.group_channels);
+        // console.log("chan " + chan);
+        // let channel_id = chan.id;
+        // console.log ("chan.id " + chan.id);
+        // console.log ("channel_id " + channel_id);
+
+        let channel_idd = "2";
+        // let chat_id = null;
+        // try {
+        //     let cur = $(this.refs.elt);
+
+        //     while (cur && cur[0].nodeName !== 'BODY') {
+        //         chat_id = cur.attr('data-chat-id');
+        //         if (chat_id) {
+        //             break;
+        //         }
+        //         cur = cur.parent();
+        //     }
+        // } catch (e) {
+        //     console.error(e);
+        // }
+
+        console.log("just 2 string " + channel_idd);
+        popover({
+            elt: (<ChatDetails chatChannelId={channel_idd} />),
+            below: this.refs.elt,
```

Comment:
replace with
```below: event.currentTarget.parentElement```

Reply:
Huh... Alright I'll give that a try, thanks :-) still confused as to why it works for player and not chat haha but thanks a lot for your help

Reply:
For Player the whole element is just the line with the player name and rank, while for Chat it's the whole chat view.

Reply:
Every element on the page is a distinct element (they are at least distinguishable by their occurrence in the html tree). And everything™ created with a tag is an [element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element).